onebusaway-android/src/androidTest/java/org/onebusaway/android/provider/test/ProviderTest.java

```diff
@@ -140,7 +140,15 @@ public void testLimit() {
                 null, null, null
         );
         assertNotNull(c);
-        assertEquals(c.getCount(), 1);
+        assertEquals(1, c.getCount());
         c.close();
+
+        //
+        // Delete
+        //
+        int result = cr.delete(uri, null, null);
+        assertEquals(1, result);
```

Comment:
The `stop_Id2` in line 121 should probably be changed to `1_1010101-TEST` to avoid a possible collision with a real stop (I know you didn't change this, but we should probably fix it while we're here).

Reply:
:+1: